client/verta/tests/unit_tests/internal_utils/test_kafka_utils.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+"""Unit tests for Kafka utilities."""
+
+from typing import Any, Dict, List
+
+import pytest
+
+from verta._internal_utils import kafka
+
+
+@pytest.fixture
+def mock_kafka_configs_response() -> Dict[str, Any]:
+    """
+    Provide mocked API result from `api/v1/uac-proxy/system_admin/listKafkaConfiguration`
+    with a single Kafka configuration.
+    """
+    return {
+        "configurations": [
+            {
+                "id": "1232abcdefg4-567h-891i-0jkl-1m1n1o2131pq",
+                "kerberos": {
+                    "enabled": True,
+                    "conf": (
+                        "[libdefaults]\ndefault_realm ="
+                        " EXAMPLE.COM\n\n[realms]\nEXAMPLE.COM"
+                        " = {\n    kdc_ports = 12,345\n    kadmind_port = 123\n    kdc ="
+                        " testing--kafka-server-X.integrations--kafka-server-headless.ci.svc.cluster.local\n"
```

Comment:
I think this kind of reveals how we're running Kafka in our cluster. I'd recommend replacing these (and `"brokerAddresses"` below with maybe `kerberos.example.com`?

Reply:
Done